docs/user/ppl/functions/string.rst

```diff
@@ -150,6 +150,31 @@ Example::
     +---------------------+---------------------+
 
 
+POSITION
+------
+
+Description
+>>>>>>>>>>>
+
+Usage: The syntax POSITION(substr IN str) returns the position of the first occurrence of substring substr in string str. Returns 0 if substr is not in str. Returns NULL if any argument is NULL.
+
+Argument type: STRING, STRING, INTEGER
```

Comment:
Does `POSITION` accept integers as an argument?

Reply:
No, the position() syntax is: `POSITION(substring IN string)`

Reply:
Should this just have `Argument type: STRING, STRING` then? Instead of `Argument type: STRING, STRING, INTEGER`

Reply:
Addressed, thanks!